packages/asset-swapper/test/market_operation_utils_test.ts

```diff
@@ -759,6 +867,62 @@ describe('MarketOperationUtils tests', () => {
                 ];
                 expect(orderSources).to.deep.eq(expectedSources);
             });
+
+            it('is able to create a order from LiquidityProvider', async () => {
+                const registryAddress = randomAddress();
+                const liquidityProviderAddress = randomAddress();
+                const xAsset = randomAddress();
+                const yAsset = randomAddress();
+                const toSell = Web3Wrapper.toBaseUnitAmount(10, 18);
+
+                const [getSellQuotesParams, getSellQuotesFn] = callTradeOperationAndRetainLiquidityProviderParams(
+                    createGetMultipleSellQuotesOperationFromRates,
```

Comment:
Feels like this whole test case should be with the "sell" test cases.

Reply:
good catch, my bad. Fixing

Reply:
fixed